businessRules/donations.js

```diff
@@ -1,15 +1,17 @@
 const { v4 } = require('uuid');
 const { createDonation } = require('../models/donations');
-const { getAllIdeas } = require('../models/ideas');
-const { getIdeas } = require('../utils/storage');
+const {
+    ideaThatMatchesIdeaId,
+} = require('../validation/ideaFunctionValidation');
```

Comment:
I would do the following:
1. Rename `validation` directory to `validations`
2. Rename `ideaFunctionValidation` to `ideas` - this will follow the same naming pattern as you have for `businessRules/ideas.js`.

Choose a naming pattern and follow it everywhere to make the code more predictable.

Reply:
Done.